i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

```diff
@@ -218,166 +333,1027 @@ Ce connecteur de supervision est compatible avec n'importe quelle distribution L
 
 ### Configuration SSH
 
-Un simple utilisateur est nécessaire pour interroger le système d'exploitation Linux par SSH. Il n'est pas nécessaire d'avoir des privilèges root ou sudo.
-Il y a deux façons possibles d'effectuer la vérification SSH, soit en échangeant la clé SSH de centreon-engine au serveur cible, 
-ou en définissant votre utilisateur et votre mot de passe directement dans les macros hôtes.
+L'utilisation de ce connecteur requiert la création d'un utilisateur sur la
+ressource supervisée, lequel sera utilisé par le collecteur Centreon pour
+s'authentifier et exécuter les requêtes SSH. Les privilèges `sudo` ou `root` ne
+sont pas nécessaires, un utilisateur 'simple' est suffisant.
 
-<Tabs groupId="sync">
-<TabItem value="Exchange des Clés SSH" label="Exchange des Clés SSH">
+Deux méthodes de connexion SSH sont possibles :
```

Comment:
@omercier shouldn't we keep the procedure?

Reply:
No. A Linux admin must know how to create a user.